tests/foreman/api/test_errata.py

```diff
@@ -458,18 +496,18 @@ def cv_publish_promote(sat, org, cv, lce=None, needs_publish=True):
     return _result
 
 
-def _publish_and_wait(sat, org, cv):
+def _publish_and_wait(sat, org, cv, search_rate=5, max_tries=12):
```

Comment:
- Though function `_publish_and_wait` is for internal use only, I would suggest to use value `search_rate=1` & `max_tries=10` as per reference at L29  in [host_helplers/capsule_mixins.py::CapsuleInfo.wait_for_tasks](https://github.com/SatelliteQE/robottelo/blob/master/robottelo/host_helpers/capsule_mixins.py#L29) 
- You can pass expected values from function call like this 
`_publish_and_wait(sat, org, cv, search_rate=5, max_tries=12)`
- I don't understant how to explain the scenario but we can consider best practices, coding standards and relate here.
- Optional, non blocking.

Reply:
Thanks! I've implemented it, to properly default to expected `search_rate=1, max_tries=10`,
if no other values were to be passed, as referenced by L29 in the link you sent.

Reply:
Cool, thank @damoore044